README.md

```diff
@@ -5,38 +5,45 @@ English | [中文](./README-zh-CN.md)
 # OceanBase Developer Center (ODC)
 
 OceanBase Developer Center, or ODC for short, is an open-source, all-encompassing tool for collaborative database development and data management across various scenarios. By streamlining collaborative efforts, ODC addresses the challenges of risk management during database changes, efficient data management, and data security concerns.
+![image](https://github.com/oceanbase/odc/assets/15030999/cef8937e-904f-43a0-b5c6-22ebae10905a)
+
 
 ## 🤔 What ODC Offers and Why Choose ODC?
 
 ### Effortless SQL Development, Anytime, Anywhere
 
 - Utilizing modern web architecture, ODC provides unparalleled accessibility to your databases, allowing you to perform SQL development tasks on the go, right from your web browser.
 - ODC boasts a comprehensive and user-friendly SQL development environment that encompasses all the features you'd expect from desktop development tools, and even offers support for PL debugging.
+![image](https://github.com/oceanbase/odc/assets/15030999/1d9e32a6-f359-4286-b0be-82cf514c0883)
 
 
 ### Guard Every Change in Your SQL Development Journey 
 
 - Throughout the entire SQL development process, ODC is your steadfast companion, offering risk control at every turn. From visual object management to SQL queries, data editing, and import/export tasks, ODC comes equipped with built-in risk management tools.
 - ODC introduces project-based collaboration and change approval workflows, complemented by SQL validation rules, SQL window guidelines, and risk level identification.
+![image](https://github.com/oceanbase/odc/assets/15030999/6d90509b-91cb-465a-ac75-1604f96c9ffd)
 
 
 ### Automated Data Lifecycle Management
 
-
 - ODC facilitates data retention policies, enabling you to effortlessly purge or archive data based on specified time frames, and build a hot-cold data separation system tailored to your needs in just 5 minutes.
 - Beyond time-based data handling, ODC supports batch processing based on partitioning, allowing you to efficiently manage large datasets.
 - With ODC's SQL scheduled tasks, completing computational tasks becomes a breeze, rendering traditional CRONTAB systems obsolete.
+![image](https://github.com/oceanbase/odc/assets/15030999/c95b57fd-3d49-4e41-96f9-32f26cb2e2d3)
+
 
 ### Sensitive Data Protection in Every Scenario 
 
 - ODC's data desensitization capabilities extend to both static and dynamic scenarios, ensuring that sensitive information remains safeguarded during schema changes, SQL queries, result set exports, and data extractions.
 - Security administrators can configure sensitive data rules and desensitization algorithms, ensuring that both DBAs and developers are unable to access sensitive data.
+![image](https://github.com/oceanbase/odc/assets/15030999/56606949-e198-4d80-a484-778e78454d65)
```

Comment:
how about change the layout from top down to left right, seems too thin, while together with other pirctures.